docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -28,10 +32,11 @@ If you're unfamiliar with the project, see the {ot-spec}[spec] for more informat
 
 Elastic's integration is designed to drop into your current OpenTelemetry setup.
 We've done this by extending the "contrib" OpenTelemetry collector and adding an Elastic exporter.
-This exporter translates the OpenTelemetry trace data collected from your services to Elastic's protocol,
-before sending the data to the Elastic Stack.
+
+Before sending the data to the {stack}, the exporter translates the OpenTelemetry trace data collected from your services
+and the metrics data collected from your infrastructure and applications to Elastic's protocol.
 By extending the OpenTelemetry collector,
-no changes are needed in your instrumented services in order to begin using the Elastic Stack.
+no changes are needed in your instrumented services to begin using the {stack}.
 
 [role="screenshot"]
 image::images/open-telemetry-elastic-arch.png[OpenTelemetry Elastic architecture diagram]
```

Comment:
Should we update the diagram to include metrics?

Reply:
I don't think so. The diagram doesn't mention data types, and there's no change to the deployment architecture.